docs/source/build_network.rst

```diff
@@ -6,1461 +6,8 @@ Building Your First Network
           a basic network, see :doc:`test_network`. If you are deploying Fabric
           in production, see the guide for :doc:`deployment_guide_overview`.
```

Comment:
Thanks for doing this Chris. In the note, where it says that BYFN has been deprecated, I think we should say removed instead of deprecated? What do you think?

Reply:
I agree, change completed.  Thanks Pam.